independent-publisher-connectors/Care Quaility Comission For England/CQC-Connector.swagger.json

```diff
@@ -0,0 +1,843 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "CQC Data",
+    "description": "To retrieve regulator review reports and ratings for England",
```

Comment:
@martynlesbirel Can you please provide some more information as it should describe concisely the main purpose and value offered by your connector

Documentation - [Description](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#description)

Reply:
Description updated please review.